includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -788,6 +814,8 @@ public static function store_validation_errors( $validation_errors, $url, $args
 			update_post_meta( $post_id, '_amp_queried_object', $args['queried_object'] );
 		}
 
+		delete_transient( static::PENDING_URLS_COUNT_TRANSIENT );
```

Comment:
I believe this is not the only place where this is needed. In particular, anywhere that `wp_update_term()` is called for the `amp_validation_error` taxonomy (like `\AMP_Validation_Error_Taxonomy::handle_validation_error_update()`) should also flush this transient, right?

Reply:
Ah, good point, I was only looking for changes in the post type, and thought this central method would cover all cases. But the query is indeed relying on taxonomies as well.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3222/commits/12733e0809844b239affcb8e8808eadb86d61f66

Reply:
I think this should cover it properly now, but not 100% certain.

Reply:
@schlessera Looks like it is needed in one more place: `\AMP_Validated_URL_Post_Type::handle_validation_error_status_update()`

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3222/commits/19521867428cff20d01786c2822bc7730457c48f